aggregator/src/aggregator/aggregation_job_creator.rs

```diff
@@ -853,6 +853,15 @@ impl<C: Clock + 'static> AggregationJobCreator<C> {
 
                         // Write the aggregation jobs and report aggregations we created
                         aggregation_job_writer.write(tx, Arc::clone(&vdaf)).await?;
+                        // Unlike the case where there is no aggregation parameter, we do _not_
```

Comment:
Yes, in the heavy hitters case, we could scrub reports once we have arrived at the last level of the prefix tree. This might require some VDAF plumbing. [I made a note in #225](https://github.com/divviup/janus/issues/225#issuecomment-2057565522).

Additionally, aggregators can still garbage collect reports once they become old enough: it's the collector's responsibility to run all the collections it wants against a batch in a timely manner.

Reply:
I agree, I think we need some way for VDAFs to indicate if a given report has been collected enough times that it can't be collected again. Perhaps a function on the set of aggregation parameters that has been collected against already, if there isn't something more clever that lets us not need to read all of the aggregation parameters.